packages/runner-ct/src/app/app.tsx

```diff
@@ -49,6 +50,11 @@ const App: React.FC<AppProps> = observer(
     const [leftSideOfSplitPaneWidth, setLeftSideOfSplitPaneWidth] = React.useState(DEFAULT_LEFT_SIDE_OF_SPLITPANE_WIDTH)
     const headerRef = React.useRef(null)
 
+    const runSpec = (spec: Cypress.Cypress['spec']) => {
```

Comment:
That's why probably it will be better to move open state to global mobx, cause if we will be able to select spec outside of sidebar this will not scale.

Reply:
Ok. Let's do this when (if?) we need to, about time this gets merged up...